packages/modal/src/Modal.tsx

```diff
@@ -226,7 +229,7 @@ function Modal({
                   onClick={handleClose}
                   className={closeButton}
                   data-dismiss="modal"
-                  aria-hidden="true"
+                  tabIndex={0}
```

Comment:
We seem to be setting tabIndex twice?

Reply:
-1 on the container, 0 on the "x" close button